services/core-api/app/api/projects/project_summary/models/project_summary.py

```diff
@@ -292,15 +294,414 @@ def create_or_update_party_orgbook(cls, party_data, party_guid, existing_party):
+                'required': True,
+                'type': 'boolean',
+            },
+            'legal_land_desc': {
+                'nullable': True,
+                'type': 'string',
+            },
+            'is_legal_land_owner': {
+                'required': True,
+                'type': 'boolean',
+            },
+            'nearest_municipality': {
+                'required': True,
+                'type': 'string',
+            },
+        }
+
+        status_code = data.get('status_code')
+        ams_authorizations = data.get('ams_authorizations', None)
+        authorizations = data.get('authorizations',[])
+        contacts = data.get('contacts')
+        applicant = data.get('applicant', None)
+        agent = data.get('agent', None)
+        is_agent = data.get('is_agent', None)
+        facility_operator = data.get('facility_operator', None)
+        facility_latitude = data.get('facility_latitude', None)
+        facility_longitude = data.get('facility_longitude', None)
+        is_legal_land_owner = data.get('is_legal_land_owner', None)
+        zoning = data.get('zoning', None)
+        facility_coords_source = data.get('facility_coords_source', None)
+
+        surface_data_to_validate = data
+        if facility_latitude != None and facility_longitude != None:
+            surface_data_to_validate = {**data, 'facility_latitude': float(facility_latitude), 'facility_longitude': float(facility_longitude)}
+
+        surface_level_schema = submission_surface_level_schema if status_code == 'SUB' else draft_surface_level_schema
+
+        if zoning == False:
+            surface_level_schema |= {
+                'zoning_reason': {
+                    'required': True,
+                    'type': 'string',
+                },         
+            }
+        
+        if facility_coords_source == 'OTH':
+            surface_level_schema |= {
+                'facility_coords_source_desc': {
+                    'required': True,
+                    'type': 'string',
+                },     
+            }
+        
+        # Validate surfave level of payload
+        validate_surface_level = Validator(surface_level_schema, purge_unknown=True)
+        if not validate_surface_level.validate(surface_data_to_validate):
+            surface_level_errors = json.dumps(validate_surface_level.errors)
+            current_app.logger.info(f'Surface level validation failed with error {surface_level_errors}')
+            current_app.logger.info(data)
+            raise BadRequest(surface_level_errors)
+
+        # Validate Basic Information
+        Project.validate_project_basic_info(data)
+
+        # Validate Authorizations Involved
+        if (status_code == 'SUB' 
+            and len(ams_authorizations.get('amendments', [])) == 0 
+            and len(ams_authorizations.get('new', [])) == 0 
+            and len(authorizations) == 0):
+                raise AssertionError('Authorizations Involved not provided')
```

Comment:
I think it would be a bad request if they don't send in any authorizations with status === SUB

Reply:
This has been adjusted now to include an error in the authorization array in the error dictionary